src/constants.ts

```diff
@@ -7,6 +7,7 @@ import type Repo from './test-cases/db/github/__types__/Repo'
 // =================================================================================================
 // Define the names of the environment variables that will be retrieved.
 const envSchema = z.object({
+  YATM_DISTRO_LABEL: z.string().default(''),
```

Comment:
Why add an empty string as the default?

In the readme, it says that it's optional. Maybe use the optional operator instead of setting a default empty string?
https://zod.dev/?id=optional

Reply:
Sounds good to me [317f340](https://github.com/audrow/yatm/pull/368/commits/317f340790198d92e856187373e0642d002b8bfe)

Reply:
Thinking about this more, I think optional is the best way to go here. (Sorry that you already made the change to make it required).

The reason is that making it required will be awkward for releases that don't use ROS. We could make it more generic by calling it something like version.

The proper way to do this is to probably have a YATM core and then maybe a YATM ROS for ROS specific things.

I'm open to either making it optional, or calling it something like version.

Reply:
Ideally we should separate the distro label we affix to tickets and the ROS distro version whose docs page we want to parse. I think the latter should be a cli argument. For example: 
```bash
npx ts-node src/index.ts requirements make docs --rosdistro rolling
```
Open to suggestions on how best to implement this.

I think leaving` YATM_DISTRO_LABEL` for the ticket label is okay.

Reply:
Yeah, that would be nice. 

Okay, we can continue with `YATM_DISTRO_LABEL`. Feel free to merge this, I can't approve since I opened the PR.

Reply:
Opened a ticket so we remember https://github.com/audrow/yatm/issues/432. Will merge this in for now.